osrparse/replay.py

```diff
@@ -123,7 +123,8 @@ def parse_life_bar_graph(self, replay_data):
 
     def parse_timestamp_and_replay_length(self, replay_data):
         format_specifier = "<qi"
-        (self.timestamp, self.__replay_length) = struct.unpack_from(format_specifier, replay_data, self.offset)
+        (t, self.__replay_length) = struct.unpack_from(format_specifier, replay_data, self.offset)
+        self.timestamp = datetime.datetime(1, 1, 1) + datetime.timedelta(microseconds=t/10)
```

Comment:
Thanks for you opinion, but I do think it's part of their API and not really an implementation detail.

https://docs.python.org/3/library/datetime.html#datetime.MINYEAR

They shouldn't be changing that definition without a new major version in python.